server/handler/leaderboard_webhook.go

```diff
@@ -0,0 +1,283 @@
+package handler
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+	"github.com/samouraiworld/topofgnomes/server/models"
+	"gorm.io/gorm"
+
+	"github.com/go-chi/chi/v5"
+)
+
+func calculateNextRunAt(webhook *models.LeaderboardWebhook) time.Time {
+	now := time.Now()
+	loc := time.Local
```

Comment:
I think time.Local will depend on the server location so probably not what you want ?